tests/foreman/ui/test_contentview.py

```diff
@@ -1382,6 +1385,27 @@ def test_positive_promote_with_custom_content(session, module_org):
         assert result['Version'] == VERSION
         result = session.contentview.promote(cv_name, VERSION, lce.name)
         assert 'Promoted to {}'.format(lce.name) in result['Status']
+        # dashboard
+        session.organization.select(org_name=module_org.name)
```

Comment:
why there is a need to select org again?

Reply:
you're right it's not necessary